Doc/whatsnew/3.12.rst

```diff
@@ -622,6 +622,18 @@ New Features
   ``__dict__`` and weakrefs with less bookkeeping,
   using less memory and with faster access.
 
+* API for performing calls using
+  :ref:`the vectorcall protocol <vectorcall>` was added to the
+  :ref:`Limited API <stable>`:
+
+  * :c:func:`PyObject_Vectorcall`
+  * :c:func:`PyObject_VectorcallMethod`
+  * :const:`Py_TPFLAGS_HAVE_VECTORCALL`
+
+  Together with the previous change, this covers both the receiving and outgoing
```

Comment:
I should have mentioned this before, but this is a little harder/stranger to parse than it could be, because "receiving" and "outgoing" don't have the clearest parallelism and are not a very idiomatic match, unless these are the standard terminology used to refer to the caller and calling ends here. What about:

```suggestion
  Together with the previous change, this covers both the sending and receiving
```

or

```suggestion
  Together with the previous change, this covers both the incoming and outgoing
```

or maybe even clearer,

```suggestion
  Together with the previous change, this covers both the caller and callee
```